### PR TITLE
Update what-is-phone-system-in-office-365.md

### PR DESCRIPTION
**Pull request by tiagoroxo**
adding Advanced Deployment Guide for Teams phone

### CODE REVIEW

File: Teams/what-is-phone-system-in-office-365.md [hunk 1]
```diff
@@ -71,6 +71,15 @@ Teams Phone can be used for services and voicemail, such as:
 
 For more information about features, see [Here's what you get with Teams Phone](here-s-what-you-get-with-phone-system.md). If you're ready to get started, see [Set up Teams Phone in your organization](setting-up-your-phone-system.md).
 
+## Microsoft Teams Phone Advanced Deployment Guide
```
Comment (JohanFreelancer9) [inline Teams/what-is-phone-system-in-office-365.md:74]:
```suggestion
## Microsoft Teams Phone advanced deployment guide
```

File: Teams/what-is-phone-system-in-office-365.md [hunk 2]
```diff
@@ -71,6 +71,15 @@ Teams Phone can be used for services and voicemail, such as:
...
+
+The Plan and implement your Microsoft Teams Phone deployment guide will help you transition from your existing voice solution to ‎Microsoft‎ ‎Teams Phone‎. You'll be guided through discovery and planning phases, or you can go straight to deployment. You'll be able to configure a calling plan, ‎Operator Connect‎, ‎Teams Phone Mobile‎, ‎Direct Routing‎, caller ID, and other features.
```
Comment (JohanFreelancer9) [inline Teams/what-is-phone-system-in-office-365.md:76]:
```suggestion
The Plan and implement your Microsoft Teams Phone deployment guide will help you transition from your existing voice solution to ‎Microsoft‎ ‎Teams Phone‎. You'll be guided through the discovery and planning phases, or you can go straight to deployment. You'll be able to configure a calling plan, ‎Operator Connect‎, ‎Teams Phone Mobile‎, ‎Direct Routing‎, caller ID, and other features.
```

File: Teams/what-is-phone-system-in-office-365.md [hunk 3]
```diff
@@ -71,6 +71,15 @@ Teams Phone can be used for services and voicemail, such as:
...
+
+> [!div class="nextstepaction"]
+> [Plan and implement your Microsoft Teams Phone deployment Guide](https://aka.ms/tpguide)
```
Comment (JohanFreelancer9) [inline Teams/what-is-phone-system-in-office-365.md:79]:
```suggestion
> [Plan and implement your Microsoft Teams Phone deployment guide](https://aka.ms/tpguide)
```

File: Teams/what-is-phone-system-in-office-365.md [hunk 4]
```diff
@@ -71,6 +71,15 @@ Teams Phone can be used for services and voicemail, such as:
...
+
+> You must be assigned an admin role such as Global Reader to access advanced deployment guides in the Microsoft 365 admin center. Only admins with the Global Administrator role can use the guides to change settings in the tenant.
```
Comment (JohanFreelancer9) [inline Teams/what-is-phone-system-in-office-365.md:81]:
```suggestion
> You must be assigned an admin role such as Global Reader to access advanced deployment guides in the Microsoft 365 admin center. Only admins with the Global Administrator role can use the guides to change settings in the organization.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @tiagoroxo, kindly commit the minor suggested changes. Many thanks.

**Comment by scanum** [comment]
We appreciate any feedback that improves the content in the Microsoft docs but we haven't had any response from you so we are going to close this issue.  
 
Please feel free to re-open this issue if there is a specific area of the docs that we can improve or make better. Thank you.

**Comment by tiagoroxo** [comment]
sorry for no replying sooner, last week we have public holidays. can we re-open so I can commit it?

**Comment by JohanFreelancer9** [comment]
@scanum please re-open. This is a recent PR. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@CarolynRowe 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by CarolynRowe** [comment]
@tiagoroxo:   I would like to review the deployment guide before approving this request.  Can you provide a link?  unfortunately, we have been asked to include links to information that turned out to be misleading or that had incorrect information.  feel free to contact me outside of github.  until then, i'm adding a do not merge tag to this PR.

**Comment by CarolynRowe** [comment]
Discussed with @roykuntz and @SerdarSoysal:  There is another thread about technical concerns in the deployment guide.  I'm going to close the PR until we know the content has been fully vetted and signed off by the product team.

**Pull request closed without merging**